### PR TITLE
feat(TDI-40559): Set user agent for azurestorage components

### PR DESCRIPTION
**Pull request by dmytro-sylaiev**
**What is the current behavior?** (You can also link to an open issue here)

**What is the new behavior?**

**Please check if the PR fulfills these requirements**

- [ ] The commit message follows Talend standard
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features) ?
- [ ] The code coverage on new code >75%
- [ ] The new code does not introduce new technical issues (sonar / eslint)

**What kind of change does this PR introduce?**

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build / CI related changes
- [ ] Other... Please describe:

**Does this PR introduce a breaking change?**

- [ ] Yes
- [ ] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/components_pull-request_master/45/) [:star:](https://ci-common.datapwn.com/job/components_pull-request_master/45/) 
[Sonar results](https://sonar.datapwn.com/overview?id=org.talend.components%3Acomponents%3Adsylaiev%2FTDI-40559_add_partner_tag_for_azure_storage)

File: components/components-azurestorage/src/main/java/org/talend/components/azurestorage/utils/AzureStorageUtils.java [hunk 1]
```diff
@@ -36,6 +37,21 @@
 
     private static final I18nMessages i18nMessages = GlobalI18N.getI18nMessageProvider().getI18nMessages(AzureStorageUtils.class);
 
+    private static OperationContext talendOperationContext;
+
+    private static final String USER_AGENT_KEY = "User-Agent";
+
+    private static final String USER_AGENT_VALUE = "APN/1.0 Talend/7.1 TCOMP/0.24";
```
Comment (olhava) [inline components/components-azurestorage/src/main/java/org/talend/components/azurestorage/utils/AzureStorageUtils.java]: Is it ok that studio version is hardcoded, maybe we need approach same as here: https://github.com/Talend/tcommon-studio-se/blob/master/main/plugins/org.talend.librariesmanager/resources/java/routines/system/Constant.java ?

Comment (dmytro-sylaiev) [reply components/components-azurestorage/src/main/java/org/talend/components/azurestorage/utils/AzureStorageUtils.java]: TCOMPv0 doesn't depend on tcommon-studio

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/components_pull-request_master/87/) [:star:](https://ci-common.datapwn.com/job/components_pull-request_master/87/) 
[Sonar results](https://sonar.datapwn.com/overview?id=org.talend.components%3Acomponents%3Adsylaiev%2FTDI-40559_add_partner_tag_for_azure_storage)

**Pull request merged**